### PR TITLE
feat(download):color link on botton in dark and light mode

### PR DESCRIPTION
**Pull request by Jufaa**
## Description

Color change in the link icon.

## Validation

Light theme:
image

Dark theme:
image

## Related Issues
 Issue: [#6440](https://github.com/nodejs/nodejs.org/issues/6440)

### CODE REVIEW

**Review by ovflowd — APPROVED** [review]

**Review by AugustinMauroy — APPROVED** [review]
LGTM ! Thanks for your first contribution.

**Comment by ovflowd** [comment]
fast-tracking this as a simple fix!

File: components/Downloads/Release/LinkWithArrow.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ const LinkWithArrow: FC<PropsWithChildren<AccessibleAnchorProps>> = ({
 }) => (
   <Link {...props}>
     {children}
-    <ArrowUpRightIcon className="ml-1 inline w-3 fill-white" />
+    <ArrowUpRightIcon className="ml-1 inline w-3 fill-shadow dark:fill-white" />
```
Comment (canerakdas) [inline components/Downloads/Release/LinkWithArrow.tsx]:
```suggestion
    <ArrowUpRightIcon className="ml-1 inline w-3 fill-neutral-600 dark:fill-white" />
```

The icon color for the white theme in [Figma](https://www.figma.com/file/pu1vZPqNIM7BePd6W8APA5/Node.js?type=design&node-id=332-34678&mode=design&t=PJg8dWIFUeV6eI3N-0) is `neutral-600`, can you update it with these 🙇

**Review by canerakdas — APPROVED** [review]

**Comment by ovflowd** [comment]
@AugustinMauroy lint failing

**Comment by AugustinMauroy** [comment]
> @AugustinMauroy lint failing

I do this 🦸🏻

**Comment by Jufaa** [comment]
Sorry for the mistakes, it's my first time  😞

**Comment by AugustinMauroy** [comment]
> Sorry for the mistakes, it's my first time 😞

The problem not come from your code but from translated file ? @ovflowd should I push fix I mean format ?

**Pull request merged**